stripe/util.py

```diff
@@ -260,6 +260,15 @@ def convert_to_stripe_object(
         return resp
 
 
+def convert_to_dict(obj):
```

Comment:
Minor note that if you send a list into `convert_to_dict`, it'll convert it to a list, not a dict.

I'd consider something like `convert_to_data`, but overall that probably reduces clarity, so `convert_to_dict` is probably our best bet.

Reply:
Yeah, I initially named the method `convert_to_python_types`, but I think `convert_to_dict` is clearer since users are unlikely to pass anything but a `StripeObject` to this method.

Reply:
Yeah, good call. +1.